### DISCUSSION TITLE
AI & ML Resources

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#79] [General help]
What resources have you found useful while learning about AI and ML? Share the name, short description, type, and a link if available.

**Example**
-[Machine Learning](https://www.techtarget.com/searchenterpriseai/definition/machine-learning-ML). This article gives an overview of what Machine Learning is, the types of ML, and more. It includes links to explainer videos to help you understand the concepts better.

**Comment by ineelhere** [comment]
Hello everyone!👋 Sharing a project I am building - **CURATED**, which can help as a great resource in your ML with Python programming journey along with some other important tech and non-tech skills. Here it is - **https://curated.streamlit.app/**.

Curated is built with the goal of helping individuals on their learning journey by providing a community-driven and well-researched reference point for discovering valuable resources. The aim here is to inform and guide, not to teach (that is more of a self defined process for every individual).

However, creating such a project requires a collective effort and input from multiple individuals. That's where you come in - as an open-source contributor, your knowledge and expertise can help make Curated even better. If this seems interesting, please feel free to collaborate and contribute on the Github repository - https://github.com/ineelhere/curated 🙂

Best wishes 🤗

**Reply by BekahHW** [reply]
Thanks for sharing!

**Reply by bimbowande** [reply]
Thanks for sharing

**Comment by BekahHW** [comment]
## Neural Networks
- [But what is a neural network?](https://www.youtube.com/watch?v=aircAruvnKk)
